presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -112,18 +113,28 @@
     private final ThriftHiveMetastoreStats stats;
     private final HiveCluster clientProvider;
     private final Function<Exception, Exception> exceptionMapper;
+    private final double scaleFactor;
```

Comment:
Please rename to `backoffScaleFactor`, so it's consistent with the config and the usage is clear (since the class is very large)

Reply:
ok